js/pro/deribit.js

```diff
@@ -0,0 +1,843 @@
+        const currency = this.safeString (params, 'currency', 'any');
+        const interval = this.safeString (params, 'interval', 'raw');
+        const kind = this.safeString (params, 'kind', 'any');
+        params = this.omit (params, 'interval', 'currency', 'kind');
+        const channel = 'user.orders.' + kind + '.' + currency + '.' + interval;
+        const message = {
+            'jsonrpc': '2.0',
+            'method': 'private/subscribe',
+            'params': {
+                'channels': [ channel ],
+            },
+            'id': this.requestId (),
+        };
+        const request = this.deepExtend (message, params);
+        const orders = await this.watch (url, channel, request, channel, request);
+        if (this.newUpdates) {
+            limit = orders.getLimit (symbol, limit);
+        }
+        return this.filterBySymbolSinceLimit (orders, symbol, since, limit, true);
+    }
+
+    handleOrders (client, message) {
+        // Does not return a snapshot of current orders
+        //
+        //     {
+        //         jsonrpc: '2.0',
+        //         method: 'subscription',
+        //         params: {
+        //             channel: 'user.orders.any.any.raw',
+        //             data: {
+        //                 web: true,
+        //                 time_in_force: 'good_til_cancelled',
+        //                 replaced: false,
+        //                 reduce_only: false,
+        //                 profit_loss: 0,
+        //                 price: 50000,
+        //                 post_only: false,
+        //                 order_type: 'limit',
+        //                 order_state: 'open',
+        //                 order_id: '46094375191',
+        //                 max_show: 10,
+        //                 last_update_timestamp: 1655401625037,
+        //                 label: '',
+        //                 is_liquidation: false,
+        //                 instrument_name: 'BTC-PERPETUAL',
+        //                 filled_amount: 0,
+        //                 direction: 'sell',
+        //                 creation_timestamp: 1655401625037,
+        //                 commission: 0,
+        //                 average_price: 0,
+        //                 api: false,
+        //                 amount: 10
+        //             }
+        //         }
+        //     }
+        //
+        if (this.orders === undefined) {
+            const limit = this.safeInteger (this.options, 'ordersLimit', 1000);
+            this.orders = new ArrayCacheBySymbolById (limit);
+        }
+        const params = this.safeValue (message, 'params', {});
+        const channel = this.safeString (params, 'channel', '');
+        const data = this.safeValue (params, 'data', {});
+        let orders = [];
+        if (this.isArray (data)) {
+            orders = this.parseOrders (data);
+        } else {
+            const order = this.parseOrder (data);
```

Comment:
same here (amount is the cost): 
![image](https://user-images.githubusercontent.com/43336371/195861877-6bced599-63b2-4aaf-9334-1161efdb6e42.png)

Reply:
Same as above, I believe it makes sense to open a seperate PR to solve the functions of `parseOrder` and `parseTrade` in the deribit base exchange